src/main/java/uk/gov/cshr/controller/IdentityController.java

```diff
@@ -3,16 +3,119 @@
 import org.springframework.security.core.Authentication;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RequestMethod;
-import org.springframework.web.bind.annotation.RestController;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+import uk.gov.cshr.domain.Identity;
+import uk.gov.cshr.domain.Role;
 import uk.gov.cshr.dto.IdentityDTO;
+import uk.gov.cshr.service.AuthenticationDetails;
 import uk.gov.cshr.service.security.IdentityDetails;
+import uk.gov.cshr.repository.IdentityRepository;
+import uk.gov.cshr.service.IdentityService;
+import uk.gov.cshr.service.RoleService;
 
-@RestController
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.Optional;
+import java.util.Set;
+
+@Controller
+@RequestMapping("/management/")
 public class IdentityController {
 
+    private static final Logger LOGGER = LoggerFactory.getLogger(RoleController.class);
+
+    @Autowired
+    private IdentityService identityService;
+
+    @Autowired
+    private RoleService roleService;
+
+    @Autowired
+    private IdentityRepository identityRepository;
+
+    @Autowired
+    private AuthenticationDetails authenticationDetails;
+
     @RequestMapping(value = "/identity", method = RequestMethod.GET)
     public IdentityDTO getIdentityDetailsfromAccessToken(Authentication authentication) {
         IdentityDetails identityDetails = (IdentityDetails) authentication.getPrincipal();
         return new IdentityDTO(identityDetails.getIdentity());
     }
+
+    @GetMapping("/identities")
+    public String identities(Model model) {
+        LOGGER.info("Listing all identities");
+
+        Iterable<Identity> identities = identityService.findAll();
+        model.addAttribute("identities", identities);
+        return "identityList";
+    }
+
+    @GetMapping("/identities/update/{uid}")
+    public String identityEdit(Model model,
```

Comment:
Maybe rename to identityUpdate for consistency?

Reply:
Yup